packages/dx-chart-core/src/plugins/zoom-pan/computeds.ts

```diff
@@ -171,3 +171,19 @@ export const detachEvents = (node: any, handlers: EventHandlers) => {
     node.removeEventListener(el, handlers[el]);
   });
 };
+
+/** @internal */
+export const getRect = (
+  interactionWithArguments, interactionWithValues, prevCoords, nextCoords, rootRef, offset,
+) => {
+  const bbox = rootRef.current.getBoundingClientRect();
```

Comment:
Can't we take `(0, 0)` as origin and `layouts.pane` as size for full rectangle?

The other thing (if `getBoundingClientRect` is still required) - `this.offset` is cached and `getBoundingClientRect` is not while they actually have same basis. Either both should be cached or both should not cached.

Reply:
`layout.pane` provide incomplete information, only width and height. We need also `left` and `top` fields

Reply:
Left and top are both `0` - there are already similar assumptions in **Axis** grids.